processor/spanprocessor/README.md

```diff
@@ -96,5 +97,18 @@ span/to_attributes:
         - ^\/api\/v1\/document\/(?P<documentId>.*)\/update$
 ```
 
+### Set status for span
+
+Example:
+
+```yaml
+# Set status allows to set specific status for a given span.
+span/set_status:
+  status:
+    code: 2
```

Comment:
It would be nicer to use symbolic status code names (Unset, Ok, Error) instead of numeric codes.